vm/vmExpr.go

```diff
@@ -560,6 +560,19 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
 		}
 		return rhsV, nil
 
+	case *ast.IfInvalidOpExpr:
+		lhsV, err := invokeExpr(e.Lhs, env)
+		isZeroValue := reflect.DeepEqual(lhsV.Interface(), reflect.Zero(lhsV.Type()).Interface())
```

Comment:
Can this be done without using DeepEqual?

Reply:
I'll have to look into this.